builder/builder.go

```diff
@@ -100,7 +102,12 @@ func (b *Builder) CreateDefaultConfig(localrpms bool) error {
 		return err
 	}
 
-	builderconf := filepath.Join(pwd, "builder.conf")
+	// Create mixer work dir (for builder.conf) if needed
+	if err = os.MkdirAll(filepath.Join(pwd, ".mixer"), 0777); err != nil {
```

Comment:
Should ".mixer" here not be b.MixerDir

Reply:
Yes, it should. And... I thought it was. I think this got lost in my pretty extensive rebasing yesterday.